docs/release/0.3.0/release-note-0.3.0.md

```diff
@@ -0,0 +1,31 @@
+# Release Note 0.3.0
+
+## ONE-vscode
+
+### OneExplorerView
+
+- Support more commands
+  - Open with Text Editor Command
+  - Open Containing Folder Command
+  - Collapse All Command
+- Improve Create Config Command
+- Prevent errors on soft link / virtualenv .cfg file
+- Rename related files automatically on cfg edit
```

Comment:
Could we surround menu name with `[ .. ]` or `"..."` like the following?

```suggestion
  - [Open with Text Editor] Command
  - [Open Containing Folder] Command
  - [Collapse All] Command
- [Improve Create Config] Command
- Prevent errors on soft link / virtualenv .cfg file
- Rename related files automatically on cfg edit
```

Reply:
I will update it. Thanks!